jetty-jaspi/src/main/config/modules/jaspi-demo.mod

```diff
@@ -0,0 +1,16 @@
+# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Enables JASPI basic authentication the /ctx context path.
```

Comment:
```suggestion
Enables JASPI basic authentication the /test context path.
```

Reply:
Looks like this will need to be fixed in 11 as well.

Reply:
Fixed in this PR. I will fix in 11 in the merge.